tests/integration/embed/embed_test.go

```diff
@@ -206,11 +206,15 @@ func setupEmbedCfg(cfg *embed.Config, curls []url.URL, purls []url.URL) {
 	cfg.ClusterState = "new"
 	cfg.ListenClientUrls, cfg.AdvertiseClientUrls = curls, curls
 	cfg.ListenPeerUrls, cfg.AdvertisePeerUrls = purls, purls
-	cfg.InitialCluster = ""
-	for i := range purls {
-		cfg.InitialCluster += ",default=" + purls[i].String()
+
```

Comment:
```suggestion
builder := new(strings.Builder)
for i, purl := range(purls) {
    if i == 0 {
        builder.WriteString(fmt.Sprintf("default=%s",purl.String()))
    } else {
        builder.WriteString(fmt.Sprintf(",default=%s",purl.String()))
    }
}
cfg.InitialCluster = builder.String()
```

Reply:
Thanks, personally I prefer the previous version of mine.
It's consistent with other code and faster.